### DISCUSSION TITLE
react-simple-maps + visx pattern + visx legend

### DISCUSSION THREAD

**Discussion by CandyDeck** [airbnb/visx#1873] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/ad0eacab-0bfd-43b2-8182-78deeb86eec0)
Hi,
I was looking for a way to implement patterns in a react-simple-maps..and I found thanks to visx pattern ! Thanks! 
However, I would like to include a legend so we know to which region are associated the patterns. 
Is there an easy way to do this ? 
Thanks !

**Comment by williaster** [comment, marked as answer]
Hey @CandyDeck thanks for checking out `visx`. have you checked out the `@visx/legend` package? https://airbnb.io/visx/docs/legend
